hoomd/md/constrain.py

```diff
@@ -279,18 +113,18 @@ class rigid(ConstraintForce):
     orientation quaternion defines the rotation from the body space into the
     simulation box. In body space, the center of mass of the body is at 0,0,0
     and the moment of inertia is diagonal. You specify the constituent particles
-    to :py:class:`rigid` for each type of body in body coordinates. Then,
-    :py:class:`rigid` takes control of those particles, and sets their position
+    to :py:class:`Rigid` for each type of body in body coordinates. Then,
```

Comment:
I believe we are changing how these are done. This occurs in a lot of places throughout this file.
```suggestion
    to `Rigid` for each type of body in body coordinates. Then,
```